images/.hadolint.yaml

```diff
@@ -6,6 +6,8 @@ ignored:
   - DL3006
   # Pin versions in apt get install: https://github.com/hadolint/hadolint/wiki/DL3008
   - DL3008
+  # Avoid additional packages by specifying --no-install-recommends: https://github.com/hadolint/hadolint/wiki/DL3015
+  - DL3015
```

Comment:
Can we not add this globally?  I had fixed this in a previous PR.  If you must, you can exempt it from specific Docker files: https://github.com/hadolint/hadolint#inline-ignores

Reply:
yes definitely, just focusing first on making the system boot before looking at any other comments/issues in here